docker/Dockerfile-workers

```diff
@@ -30,6 +30,7 @@ RUN rpm -e --nodeps python3-amqp && pip3 install amqp==5.0.2
 
 # All the requirements except pyarn should already be installed
 RUN pip3 install pyarn && \
+    RUN pip3 install -r requirements.txt --no-deps --require-hashes
```

Comment:
I think we no longer have to install `pyarn` separately, it is in the requirements file.